build-system/compile/bundles.config.extensions.json

```
@@ -1216,6 +1216,14 @@
     "version": "0.1",
     "latestVersion": "0.1"
   },
+  {
+    "name": "amp-wordpress-embed",
+    "version": "1.0",
+    "latestVersion": "1.0",
+    "options": {
+      "wrappers": "bento"
```

The entry for your component must have `options.wrapper = "bento"`, and may optionally include `options.npm = true` to opt the Preact component portion for automated npm publishing on AMP's release schedule.
See: https://github.com/ampproject/amphtml/pull/34908
```suggestion
      "npm": true,
      "wrappers": "bento"
```